README.rst

```diff
@@ -77,6 +77,29 @@ Remember to add a ``SECRET_KEY`` setting!
     python manage.py index
 
 
+Static Files
+------------
+
+Static files are stored in `sitemedia/` and bundled by Webpack. The `webpack-bundle-tracker` plugin generates a JSON manifest file listing the name and location of bundled files. This file — `webpack-stats.json` in production/QA/test and `webpack-stats-dev.json` for local development — is read by Django using `django-webpack-loader` so that the relevant files can be included in the template's `<head>`.
+
+Bundled files and the production JSON manifest are checked into git and will be collected by Django's `collectstatic` command. They should be recompiled and any changes should be committed as part of a release. To recompile bundles for production::
+
+    npm run build
+
+When actively developing stylesheets and scripts, you can instead run a development Webpack server with live reload capability. To do this, first uncomment or add the line to your `local_settings.py` that tells Django to read from the development Webpack manifest instead::
+
+    WEBPACK_LOADER["DEFAULT"]["STATS_FILE"] = BASE_DIR.parent / "sitemedia" / "webpack-stats-dev.json"
+
+Then, run a development server to generate the manifest::
+
+    npm start
```

Comment:
When I ran this I was prompted to install `webpack-dev-server`; should that be included in npm packages ?

Reply:
That's odd; it worked for me without installing, but I couldn't tell if that package was automatically installed along with `webpack` or not. I'll add it to `package.json` regardless